vpr/src/base/read_options.cpp

```diff
@@ -1378,6 +1378,10 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .help("Displays delay statistics even if placement is not timing driven")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    place_grp.add_argument<bool, ParseOnOff>(args.use_expansion_for_delay_matrix, "--use_expansion_for_delay_matrix")
```

Comment:
I'd suggest changing the definition of this option to something like: '--place_delta_delay_matrix_calculation_method {astar, dijkstra}'.

That seems to better capture what this option is trying to do (and leaves the door open for other methods to be added in the future).

Reply:
Done